linodecli/plugins/obj.py

```diff
@@ -613,6 +616,13 @@ def call(args, context):
                 message = e.message
             print('Error: {}'.format(message))
             sys.exit(6)
+    elif parsed.command == "regenerate-keys":
+        print("Regenerating Object Storage keys..")
+        _get_s3_creds(context.client, force=True)
+        print("Done.")
+        print("Warning: Your old Object Storage keys _were not_ expired!  If you want "
```

Comment:
```suggestion
        print("Warning: Your old Object Storage keys _were not_ expired automatically!  If you want "
```
Just to avoid any confusion in case the keys actually were expired and that's why they were changing them.